test/screenshot/mdc-fab/classes/extended.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2018 Google Inc. All rights reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Extended FAB (Floating Action Button) - MDC Web Screenshot Test</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="../../out/fixture.css">
+    <link rel="stylesheet" href="../../out/mdc.fab.css">
+    <link rel="stylesheet" href="../../out/mdc-fab/fixture.css">
+  </head>
+
+  <body class="test-body">
+    <main class="test-main test-main--fab">
+      <div class="test-grid">
+        <div class="test-cell">
```

Comment:
We should be explicitly specifying a width for `test-cell`; see other screenshot tests for examples. (Usually it's a multiple of 10 + 1 to show the grid lines on both ends, e.g. 81px.)